resources/c/partial/submissions.sass

```diff
@@ -38,3 +38,7 @@
   
 .submissions-list
   list-style: circle
+  
+.sub-heading
+  font-family: Vanadine
+  font-size: 25px
```

Comment:
Set font-size in rem rather than px, so that it scales in proportion on different sized viewports.